LogGenerator/src/main/java/com/loggenerator/LogGenerator.java

```diff
@@ -0,0 +1,166 @@
+        "Henry", "Isabella", "Jacob", "Kate", "Liam", "Mia", "Nathan",
+        "Olivia", "Peter", "Quinn", "Rose", "Sarah", "Tom", "Uma",
+        "Victor", "Wendy", "Xander", "Yara", "Zack", "Adam", "Bella",
+        "Cameron", "Diana", "Ethan", "Fiona", "Gavin",
+        "Hazel", "Ian", "Julia", "Kai", "Lila", "Mason", "Nora",
+        "Owen", "Piper", "Quincy", "Riley", "Sofia", "Tyler", "Violet",
+        "Wyatt", "Ximena", "Yvette", "Zara"
+    };
+    private static final String[] MESSAGESWCPF = {"O usuario {CPF} realizou {ACTION} em {DATETIME}.",
+        "Um novo usuario {CPF} foi criado em {DATETIME}.",
+        "O perfil do usuario {CPF} foi atualizado em {DATETIME}.",
+        "O produto com ID {ID} foi adicionado ao carrinho pelo usuario {CPF} em {DATETIME}.",
+        "O usuario {CPF} finalizou sua compra em {DATETIME}."};
+
+    private static final String[] MESSAGES = {"O usuario {CPF} realizou {ACTION} em {DATETIME}.",
+        "Um novo usuario {CPF} foi criado em {DATETIME}.",
+        "O usuario {CPF} fez login em {DATETIME}.",
+        "O usuario {CPF} fez logout em {DATETIME}.",
+        "O perfil do usuario {CPF} foi atualizado em {DATETIME}.",
+        "O usuario {CPF} deletou sua conta em {DATETIME}.",
+        "O produto com ID {ID} foi adicionado ao carrinho pelo usuario {CPF} em {DATETIME}.",
+        "O usuario {CPF} finalizou sua compra em {DATETIME}.",
+        "O usuario {CPF} enviou uma mensagem para o usuario {TO_USER} em {DATETIME}.",
+        "O usuario {CPF} compartilhou {CONTENT} com seus seguidores em {DATETIME}.",
+        "O usuario {CPF} fez o download de {FILE_NAME} em {DATETIME}.",
+        "Um novo produto foi adicionado ao inventario em {DATETIME}.",
+        "Um pedido foi feito pelo cliente {CUSTOMER_ID} em {DATETIME}.",
+        "Um pagamento de {AMOUNT} foi processado em {DATETIME}.",
+        "Um novo post de blog foi publicado em {DATETIME}.",
+        "Um novo evento foi agendado para {DATE} as {TIME}.",
+        "Uma nova candidatura de emprego foi recebida em {DATETIME}.",
+        "Uma nova mensagem foi recebida de {SENDER_NAME} em {DATETIME}.",
+        "Uma nova assinatura foi criada por {CUSTOMER_NAME} em {DATETIME}.",
+        "Um novo arquivo foi enviado para o servidor em {DATETIME}.",
+        "Uma nova venda foi criada em {DATETIME}.",
+        "Um novo plano de assinatura foi criado em {DATETIME}.",
+        "Uma nova vaga de emprego foi publicada em {DATETIME}.",
+        "Um novo relatorio de erro foi enviado em {DATETIME}.",
+        "Uma nova avaliacao de produto foi enviada por {USER_NAME} em {DATETIME}.",
+        "Uma nova inscricao em evento foi recebida de {USER_NAME} em {DATETIME}.",
+        "Uma nova consulta foi agendada para {DATE} as {TIME}."};
+
+    private static String generateDate(Random random) {
+        Calendar calendar = Calendar.getInstance();
+
+        // Gerando ano aleatorio entre 1950 e 2023
```

Comment:
Could you please use english in the comments + code, just so that we are consistent throughout?